spec_legacy/unit/lib/redmine/scm/adapters/git_adapter_spec.rb

```diff
@@ -42,8 +42,8 @@
   ## Ruby uses ANSI api to fork a process on Windows.
   ## Japanese Shift_JIS and Traditional Chinese Big5 have 0x5c(backslash) problem
   ## and these are incompatible with ASCII.
-  # WINDOWS_PASS = Redmine::Platform.mswin?
-  WINDOWS_PASS = false
+  # WINDOWS_PASS1 = Redmine::Platform.mswin?
+  WINDOWS_PASS1 = false
```

Comment:
Yep, and since this legacy specs, did not bother to give it a more meaningful name.